cmake/sysbuild/suit.cmake

```diff
@@ -317,6 +331,50 @@ function(suit_setup_merge)
   endforeach()
 endfunction()
 
+# Build recovery image.
+#
+# Usage:
+#   suit_build_recovery()
+#
+function(suit_build_recovery)
+  include(ExternalProject)
+
+  set(sysbuild_root_path "${ZEPHYR_NRF_MODULE_DIR}/../zephyr/share/sysbuild")
+
+  ExternalProject_Add(
+    recovery
+    SOURCE_DIR ${sysbuild_root_path}
+    PREFIX ${CMAKE_BINARY_DIR}/recovery
+    INSTALL_COMMAND ""
+    CMAKE_CACHE_ARGS
+      "-DAPP_DIR:PATH=${ZEPHYR_NRF_MODULE_DIR}/samples/suit/recovery"
+      "-DBOARD:STRING=nrf54h20dk/nrf54h20/cpuapp"
```

Comment:
`nrf54h20dk` cannot be assumed. See https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/suit.cmake#L16 to see how to make the target string.

Reply:
Ok fixed it. I didn't go for exactly the same solution, however the DK is no longer assumed.